lib/qx/tool/cli/commands/Compile.js

```diff
@@ -472,6 +472,10 @@ qx.Class.define("qx.tool.cli.commands.Compile", {
           });
         }
       });
+
+      if (!data.libraries.every(libData => fs.existsSync(libData + "/Manifest.json"))) {
+        await (new qx.tool.cli.commands.contrib.Install({quiet:true})).process();        
```

Comment:
Maybe inform the user about implicit installation unless a "quiet" flag has been set?

Reply:
we have no message in auto update and list either. Should'nt we inform the user in this case too?

Reply:
Done